### PR TITLE
Incorporate reporting of emissions from agricultural residue burning

### PR DESCRIPTION
**Pull request by mscrawford**

### CODE REVIEW

File: R/NitrogenBudget.R [hunk 1]
```diff
@@ -41,18 +41,18 @@ NitrogenBudget<-function(gdx,level="reg",dir=".",spamfiledirectory="",debug=FALS
...
     seed_detail <- Seed(gdx,level=level,attributes = "nr")
     seed <- dimSums(Seed(gdx,level=level,attributes = "nr"),dim=3)
-    
+
     ag_recycling <-dimSums(readGDX(gdx,"ov18_res_ag_recycling",select=list(type="level"))[,,"nr"],dim=c(3.1,3.2))
     ag_recycling <- gdxAggregate(gdx = gdx,weight = 'ResidueBiomass',x = ag_recycling, to = level,absolute = TRUE,dir = dir, product_aggr = T,attributes = "nr",plantpart="ag")
-    
-    ash<- dimSums((readGDX(gdx,"ov18_res_ag_burn",select=list(type="level"))[,,kcr]*(1-readGDX(gdx,"f18_res_combust_eff")[,,kcr]))[,,"nr"],dim=3)
+
+    ash <- dimSums((readGDX(gdx,"ov_res_ag_burn","ov18_res_ag_burn",select=list(type="level"))[,,kcr]*(1-readGDX(gdx,"f18_res_combust_eff")[,,kcr]))[,,"nr"],dim=3)
```
Comment (tscheypidi) [inline R/NitrogenBudget.R:48]: I believe it must be `readGDX(..., format="first_found")` (you want to get either ov_res_ag_burn or ov18_res_ag_burn), right?

File: R/ResidueUsage.R [hunk 2]
```diff
@@ -15,105 +15,105 @@
...
   kcr2kres <- readGDX(gdx,"kres_kcr")
-  
+
   if(level%in%c("reg","regglo","glo")){
     ResidueBiomass <- collapseNames(readGDX(gdx,"ov_res_biomass_ag")[,,"level"][,,"dm"])
-   
+
     Recycling_kcr  <- collapseNames(readGDX(gdx,"ov18_res_ag_recycling")[,,"level"][,,"dm"])
-    Burn_kcr       <- collapseNames(readGDX(gdx,"ov18_res_ag_burn")[,,"level"][,,"dm"])
+    Burn_kcr       <- collapseNames(readGDX(gdx,"ov_res_ag_burn","ov18_res_ag_burn")[,,"level"][,,"dm"])
```
Comment (tscheypidi) [inline R/ResidueUsage.R:38]: same as above

File: DESCRIPTION [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Package: magpie4
 Type: Package
 Title: MAgPIE outputs R package for MAgPIE version 4.x
-Version: 1.86.7
-Date: 2021-02-16
+Version: 1.87.0
+Date: 2021-02-18
 Authors@R: c(person("Benjamin Leon", "Bodirsky", email = "bodirsky@pik-potsdam.de", role = c("aut","cre")),
```
Comment (tscheypidi) [inline DESCRIPTION:6]: don't forget to add your name to the list of authors (forgot to mention that in mrcommons as well)

Comment (tscheypidi) [reply DESCRIPTION:6]: also add your name in the author information of the functions to modify

**Review by tscheypidi — CHANGES_REQUESTED** [review]

**Comment by mscrawford** [comment]
To follow up on this, I had to merge my changes into Abhi's subsequent changes. I added first_found to the readGDX calls as well as my name to the author list.

**Review by tscheypidi — APPROVED** [review]

**Pull request merged**